modules/edu.gemini.seqexec.engine/src/main/scala/edu/gemini/seqexec/engine/state.scala

```diff
@@ -0,0 +1,155 @@
+package edu.gemini.seqexec.engine
+
+import scalaz._
+
+/**
+ * Flag to indicate whether the global execution is `Running` or `Waiting`.
+ */
+sealed trait Status
+
+object Status {
```

Comment:
Is this status only for for the global state or will it be used at other levels like sequence?